vmdb/app/assets/javascripts/miq_application.js

```diff
@@ -1162,7 +1169,7 @@ function miqBuildCalendar(bAngular) {
     }
     if (typeof miq_cal_skipDays != "undefined" &&
         miq_cal_skipDays != null &&
-        miq_cal_skipDays != '') {
```

Comment:
@martinpovolny 
This is same case as with `counter` variable...
Actually this conditions isn't probably usefull here, since `miq_cal_skipDays` is tested on `null`